### PR TITLE
feat: add version control to documentation

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Resolve https://github.com/oras-project/oras-www/issues/317. Add version control to ORAS documentation. ORAS users will be able to choose a version from the drop-down list.

v1.2.0 is the default version. `Next` is the upcoming version which is in development. 

image

### CODE REVIEW

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: versioned_docs/version-1.1/community/_category_.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "label": "Community",
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply versioned_docs/version-1.1/community/_category_.json:2]: In general, all docs under this folder will be versioned and moved to a dedicated versioned folder https://github.com/oras-project/oras-www/tree/main/docs. Are there any concerns of versioning the community docs along with other docs? If YES, we may need to move this chapter to another place.

[the remark this answers is not in the document]
Comment (TerryHowe) [reply versioned_docs/version-1.1/community/_category_.json:2]: The community docs should be the same for all versions

**Comment by TerryHowe** [comment]
All the links in the footer link to unversion pages and seem to be broken

**Comment by TerryHowe** [comment]
I don't see links to the community pages in the preview

**Comment by FeynmanZhou** [comment]
> I don't see links to the community pages in the preview



> Details

@TerryHowe I temporarily hide the doc links within footer section in this PR. I may need more time to investigate and will resolve it in the next PR.

**Comment by FeynmanZhou** [comment]
> I don't see links to the community pages in the preview

@TerryHowe 
Do you expect that the Community pages should be existed in the current release (v1.2.0) only?
image

**Comment by TerryHowe** [comment]
All the links in the footer are missing or broken now

**Comment by TerryHowe** [comment]
I spent an hour messing with this and haven't improved anything although I suspect something around https://docusaurus.io/docs/versioning#navbar-items would fix the links. I was mostly trying to understand how to have versioned and unversioned chunks of documents, but that isn't working either

**Comment by FeynmanZhou** [comment]
> I spent an hour messing with this and haven't improved anything although I suspect something around https://docusaurus.io/docs/versioning#navbar-items would fix the links. I was mostly trying to understand how to have versioned and unversioned chunks of documents, but that isn't working either

@TerryHowe 
Thanks for your investigation.
I also looked into the doc but has not found a solution https://docusaurus.io/docs/versioning#navbar-items. 
So I temporarily hide the doc links within footer section. I also moved the Community page from "Next" to the current version "v1.2.0".

I may need more time to investigate the versioned link route and will resolve it in the next PR.

**Pull request merged**